lib/attack.go

```diff
@@ -229,6 +229,15 @@ func Client(c *http.Client) func(*Attacker) {
 	return func(a *Attacker) { a.client = *c }
 }
 
+// ProxyHdr returns a functional option that allows you to add your own
+// Proxy CONNECT headers
+func ProxyHdr(h *http.Header) func(*Attacker) {
```

Comment:
Please call this `ProxyHeader`

Reply:
You can pass a non-pointer value since the `http.Header` type is a reference type: map.